packages/fx-core/src/plugins/resource/frontend/ops/deploy.ts

```diff
@@ -57,7 +58,10 @@ export class FrontendDeployment {
 
     await progressHandler?.next(DeploySteps.Build);
     await runWithErrorCatchAndThrow(new BuildError(), async () => {
-      await Utils.execute(Commands.BuildFrontend, componentPath);
+      await Utils.execute(Commands.BuildFrontend, componentPath, {
+        ...envs.customizedRemoteEnvs,
```

Comment:
I find the second parameter of Utils.execute is workingDir, which confused me with this env value, packages/fx-core/src/plugins/resource/frontend/utils.ts